packages/plugins/plugin-sheet/src/components/Toolbar/Toolbar.tsx

```diff
@@ -214,7 +219,7 @@ const Alignment = () => {
             ({ range, key }) => key === alignKey && inRange(rangeFromIndex(model.sheet, range), cursor),
           )?.value
         : undefined,
-    [cursor, model.sheet.ranges],
```

Comment:
this doesn't need to be recreated if the ranges change?

Reply:
I think I was trying to figure out reactivity here, will revert and create a new issue to address toolbar reactivity.